db/migrate/20230522135216_add_index_to_topics.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToTopics < ActiveRecord::Migration[7.0]
+  def change
+    add_index :topics, :name, unique: true
```

Comment:
This can be added in the other migration file and have only one, wdyt?

Reply:
Done!